File: TEST data for large repo/Repo_copy/Blueprism/BluePrism.Api/BluePrism.Api.Models/ResourcesSummaryUtilizationParameters.cs
namespace BluePrism.Api.Models
{
    using System;
    using System.Collections.Generic;

    public class ResourcesSummaryUtilizationParameters
    {
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public IReadOnlyCollection<Guid> ResourceIds { get; set; }
    }
}

File: TEST data for large repo/Repo Copy-3/Blueprism/BluePrism.Server.Domain.Models/ResourcesSummaryUtilizationParameters.vb
Imports System.Runtime.Serialization

<Serializable()>
<DataContract([Namespace]:="bp")>
Public Class ResourcesSummaryUtilizationParameters

    <IgnoreDataMember>
    <CLSCompliantAttribute(False)>
    Public Property StartDate As DateTime
    <IgnoreDataMember>
    <CLSCompliantAttribute(False)>
    Public Property EndDate As DateTime
    <IgnoreDataMember>
    <CLSCompliantAttribute(False)>
    Public Property ResourceIds As IEnumerable(Of Guid)
   
End Class
